corehq/messaging/management/commands/import_conditional_alerts.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from corehq.apps.data_interfaces.models import (
+    AutomaticUpdateRule,
+    MatchPropertyDefinition,
+    CreateScheduleInstanceActionDefinition,
+)
+from corehq.apps.domain.models import Domain
+from corehq.messaging.management.commands.export_conditional_alerts import (
+    SimpleSchedulingRule,
+    SimpleSMSDailyScheduleWithTime,
+    SimpleSMSAlertSchedule,
+    SIMPLE_SMS_DAILY_SCHEDULE_WITH_TIME,
+    SIMPLE_SMS_ALERT_SCHEDULE,
+)
+from corehq.messaging.scheduling.models import (
+    AlertSchedule,
+    TimedSchedule,
+    TimedEvent,
+    SMSContent,
+)
+from corehq.messaging.tasks import initiate_messaging_rule_run
+from django.core.management.base import BaseCommand, CommandError
+from django.db import transaction
+from io import open as io_open
+import json
+import six
+
+
+def open_for_json_read(path):
+    # json.dumps returns bytes in python2, but unicode in python3
+    # this function mirrors open_for_json_write in export_conditional_alerts
+    if six.PY2:
+        return open(path, 'rb')
+
+    return io_open(path, 'r', encoding='utf-8')
+
+
+class Command(BaseCommand):
+
+    def add_arguments(self, parser):
+        parser.add_argument('domain')
```

Comment:
Add helptext? (this comment applies to the other parser arguments in this pr too)

Reply:
updated: https://github.com/dimagi/commcare-hq/pull/21366/commits/1314aaaf2d9542fca21bab217c932224e217371f